src/interfaces/root-state-interface.ts

```diff
@@ -15,7 +15,8 @@ export interface StateFilingIF {
 /** The state model interface for the Root Store. */
 export interface RootStateIF {
   // tombstone data
-  authRoles: Array<string>
+  accountInformation: AccountInformationIF
```

Comment:
If this is only used to identify SBC Staff users then I think this property (and all the code to fetch it, etc) will be obsolete soon. Just keep this in mind for when we update this UI to use auth roles directly.